### ISSUE TITLE
<<- assigns to closest parent environment symbol

### ISSUE THREAD

**Issue by kevinushey** [tdsmith/aRrgh#12]
E.g.

```
x <- 1

f <- function() {
  x <- 3
  g <- function() {
    x <<- 2
    return(NULL)
  }
  g()
  return(x)
}

print( c(f(), x) ) ## [1] 2 1
```

**Comment by tdsmith** [comment]
I'm kinda thinking there should be a section on namespaces and name resolution where this would fit nicely? I don't understand it yet though so I don't know when that's going to happen!

**Comment by tdsmith** [comment]
Also, Google offered to translate the text of the Github auto-email from Lithuanian. R!

**Comment by kevinushey** [comment]
This is an exceptionally good post breaking all of that down: http://obeautifulcode.com/R/How-R-Searches-And-Finds-Stuff/ - it shares the similarly frustrated-with-R tone as well :)

**Comment by tdsmith** [comment]
Oh I fixed this in 87c5b7baa2e7c565b9e6fbeedd7b71d9dc973ad8 I think!

**Comment by tdsmith** [comment]
Added that link in 381bec0
